{"question": "\nI am just getting started with Qiskit and IBM Quantum Experience, so please forgive my newby question.\nI have a IBM Quantum Experience account and I generated an API token.\nI used the generated token in the following tiny Python3 program:\nimport qiskit\nQX_CONFIG = {\n  \"APItoken\": \"<my-token-here>\",\n  \"url\":\"https://quantumexperience.ng.bluemix.net/api\"}\nqiskit.register(QX_CONFIG['APItoken'], QX_CONFIG['url'])\n\nWhen I run this program qiskit.register throws the following exception:\nException has occurred: qiskit._qiskiterror.QISKitError\n\"Couldn't instantiate provider! Error: Couldn't connect to IBMQuantumExperience server: error during login: HTTPSConnectionPool(host='quantumexperience.ng.bluemix.net', port=443): Max retries exceeded with url: /api/users/loginWithToken (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:645)'),))\"\n\nThe API endpoint is reachable:\n$ curl https://quantumexperience.ng.bluemix.net/api\n\n{\"error\":{\"status\":400,\"message\":\"Generic error\",\"code\":\"GENERIC_ERROR\"}}\n\nAny suggestions on how to fix this issue?\nPS: \napi = IBMQuantumExperience(QX_API_TOKEN)\n\ngenerates the same exception\n", "answers": [], "comments": ["That should work if you provided the right API token and url. Could you try the code from : github.com/Qiskit/qiskit-tutorial/blob/master/hello_world/\u2026", "It seems like you are doing things right. Have you tried again? The new 0.6 version of Qiskit has new options for registration, so perhaps those could also help you stave off the error."], "link": "https://quantumcomputing.stackexchange.com//questions/4324/qiskit-python-program-cannot-connect-to-qe-api?r=SearchResults"}